class.jetpack-gutenberg.php

```diff
@@ -89,6 +102,52 @@ class Jetpack_Gutenberg {
 	 */
 	private static $availability = array();
 
+	/**
+	 * Check to see if a block is able to run with current wp or gutenburg plugin versions
+	 *
+	 * @param string $core_wp_version The minimum wp core version needed to run the block.
+	 * @param string $plugin_version The minimum gutenberg plugin version needed to run the block.
+	 *
+	 * @since 8.4.0
+	 *
+	 * @return boolean True if the block is gated due to current wp or plugin versions.
+	 */
+	public static function is_block_version_gated( $core_wp_version, $plugin_version ) {
```

Comment:
This is a nitpick, but I feel something is version gated if it has the version arguments set during registration, and this method is more about checking if it meets the version requirements. I'm not sure what a better name is though `is_editor_version_available` maybe?

Reply:
yeh, as they say, naming is one of the two hardest things in computer science - I wasn't 100% happy with the naming here, so will have a think about a better alternative.

Reply:
Indeed it is! I think along with caching and off by one errors.